orchagent/intfsorch.cpp

```diff
@@ -261,7 +292,15 @@ void IntfsOrch::doTask(Consumer &consumer)
         {
             if (alias == "lo")
             {
-                removeIp2MeRoute(vrf_id, ip_prefix);
+                if (m_syncdIntfses[alias].ip_addresses.count(ip_prefix))
```

Comment:
In the set operation, you skip adding the loopback in m_syncdIntfses if the key (loopback) has no prefix. I think you will crash here.

Reply:
right.  There is also vrf_id issue to be taken care of.